airbyte-integrations/connectors/destination-s3/src/main/kotlin/io/airbyte/integrations/destination/s3/S3DestinationAsyncStrictEncrypt.kt

```diff
@@ -9,7 +9,7 @@ import io.airbyte.cdk.integrations.destination.s3.S3BaseChecks.testCustomEndpoin
 import io.airbyte.cdk.integrations.destination.s3.S3DestinationConfigFactory
 import io.airbyte.protocol.models.v0.AirbyteConnectionStatus
 
-class S3DestinationStrictEncrypt : S3Destination {
+class S3DestinationAsyncStrictEncrypt : S3DestinationAsync {
```

Comment:
Wrong rename. Should be S3destinationAsyncStrictEncrypt, if we keep the remaining.